src/krati_Jain/Assignment_16/CharacterFrequency.java

```diff
@@ -0,0 +1,37 @@
+/*Find the frequency of character from a given String array.
+Note: Take String and character from the user using scanner class.*/
+
+package krati_Jain.Assignment_16;
+
+import java.util.Scanner;
+
+public class CharacterFrequency {
+		
+	int count;
```

Comment:
You should write this instance variable with in method as, it’s useful for only that method.  If variable going to use multiple places then it makes senses to define that variable at instance. just trying to explain good coding practices.